netplan_cli/cli/commands/apply.py

```diff
@@ -309,6 +315,14 @@ def command_apply(self, run_generate=True, sync=False, exit_on_error=True, state
                         break
                     time.sleep(0.5)
 
+            # If "lo" is managed by NM through Netplan, apply will flush its addresses and NM
+            # will not bring it back automatically like other connections.
+            # This is a possible scenario with netplan-everywhere. If a user tries to change the 'lo'
+            # connection with nmcli for example, NM will create a persistent nmconnection file and emit a YAML for it.
+            if 'lo' in nm_interfaces:
+                cmd = ['nmcli', 'con', 'up', 'lo']
+                subprocess.call(cmd, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```

Comment:
nitpick: We should consider using the `utils.nmcli()` helper here.

Reply:
`utils.nmcli()` calls `check_call`, which will throw an exception if nmcli fails for some reason, maybe we should use `call` to avoid breaking `apply` (or to avoid a try-except here...).